Sources/SwiftProtobuf/UnsafeRawPointer+Shims.swift

```diff
@@ -0,0 +1,39 @@
+// Sources/SwiftProtobuf/UnsafeRawPointer+Shims.swift - Shims for UnsafeRawPointer and friends
+//
+// Copyright (c) 2019 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/master/LICENSE.txt
+//
+// -----------------------------------------------------------------------------
+///
+/// Shims for UnsafeRawPointer and friends.
+///
+// -----------------------------------------------------------------------------
+
+
+extension UnsafeRawPointer {
```

Comment:
I've been burned in the past extending standard system types like this.  Inevitably, they end up conflicting with something.

If you plan to remove these in a subsequent PR, please comment them as such.

Reply:
This extension _should_ be safe, because even if the standard library added these methods these extensions would still take priority over the standard library within the module. As the extended methods do not add _public_ API to the types, nothing bad should happen.

Nonetheless, I do want to remove these, so I've updated them with comments indicating that they are planned to be removed.

Reply:
Thanks for adding the comment.

To clarify my concern:  You're right that Swift's scoping rules should prevent this from actually breaking any code (unlike when people try this in Javascript or Obj-C), but it's still an avoidable point of confusion.  People new to this code base would likely assume these methods were the ones from the standard library and be confused by any behavioral differences.

Reply:
I totally agree. If you’d prefer not to have these shims at all I can replace all the usage sites with `store` and `load`.

Reply:
I'm okay merging this as-is as long as it's clearly commented as a wart to be removed.